book/src/dev/release-criteria.md

```diff
@@ -27,87 +27,33 @@ While Zebra is still in alpha, we don't guarantee support for any particular pla
+        - ⚠️ Acceptance tests pass reliably
 - Implementation and Launch
-    - :red_circle: All [release blocker bugs]() have been fixed
-    - :green_circle: Users can access the documentation to deploy `zebrad` nodes
+    - 🛑 All [release blocker bugs](https://github.com/ZcashFoundation/zebra/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22First+Alpha+Release%22+label%3AC-bug) have been fixed
+    - ✅ Users can access the documentation to deploy `zebrad` nodes
 - User Experience
-    - :green_circle: Build completes within 30 minutes in Zebra's CI
-    - :yellow_circle: `zebrad` executes normally
-        - :green_circle: `zebrad`'s default logging works reasonably well in a terminal
-        - :green_circle: panics, error logs, and warning logs are rare on mainnet
-        - :yellow_circle: known panics, errors and warnings have open tickets
-
-## Future Releases
-
-### System Requirements
-
-_TBD_
-
-### Build Requirements
-
-_TBD_
-
-### Supported Platforms
-
-_TBD_
-
-### Go/No-Go Criteria
-
-- `zebrad` Functionality
-    - [ ] `zebrad` can sync to mainnet tip
-        - [ ] under perfect network conditions (within 2 - 3 hours)
-        - [ ] under reasonable network conditions (in under 5 hours)
-        - [ ] under sub-optimal network conditions (?)
-    - [ ] `zebrad` can keep up with the mainnet tip after the initial sync
-        - [ ] under perfect network conditions
-        - [ ] under reasonable network conditions
-        - [ ] under sub-optimal network conditions
-    - [ ] `zebrad` can validate proof of work
-    - [ ] `zebrad` can validate the transaction merkle tree
-    - [ ] `zebrad` can serve blocks to peers
-- `zebrad` Performance
-    - [ ] `zebrad` functionality works on platforms that meet its system requirements
-- Testing
-    - [ ] CI Passes
-        - [ ] Unit tests pass reliably
-        - [ ] Property tests pass reliably
-        - [ ] Acceptance tests pass reliably
-- Network Readiness
-    - [ ] We have a mechanism to deal with misbehaving `zebrad` nodes
-- Implementation and Launch
-    - [ ] All release blocker bugs have been fixed
-    - [ ] Release notes are available
-    - [ ] Known issues are clearly documented
-    - [ ] Users can access the documentation to deploy `zebrad` nodes
-- User Experience
-    - [ ] Build completes within 20 minutes in Zebra's CI
-    - [ ] `zebrad` executes normally
-        - [ ] `zebrad`'s default logging works reasonably well in a terminal
-        - [ ] panics, error logs, and warning logs are rare
-        - [ ] known panics, errors and warnings have open tickets
-    - [ ] Users can access resources to troubleshoot `zebrad` malfunction
-- Support
-    - [ ] we have a clearly documented mechanism for users to report issues with `zebrad`
-    - [ ] we have the ability to reproduce reported issues
-    - [ ] `zebrad` reports sufficient information to allow us to troubleshoot reported issues
+    - ✅ Build completes within 30 minutes in Zebra's CI
```

Comment:
Most builds seem to complete within 30 minutes, but I've seen the occasional slow build.
Some of our build steps seem to be re-compiling things they don't need to.
```suggestion
    - ⚠️ Build completes within 30 minutes in Zebra's CI
```